config/locales/hyrax.de.yml

```diff
@@ -168,8 +168,8 @@ de:
             success: Das Design wurde erfolgreich aktualisiert
       features:
         index:
-          action: Aktion
-          description: Beschreibung
+          feature: Features
```

Comment:
Did you mean to replace the `action` and `desciption` keys with duplicate `feature` and `header` keys?

Reply:
oops, s.t. wet wrong. You are right  action and desciption keys should not change. 
 how to handle this. New PR?
  169       features:
  170         index:
  171           action: Aktion
  172           description: Beschreibung
  173           feature: Features
  174           header: Features